src/modules/connection.js

```diff
@@ -25,7 +27,7 @@ class Connection {
     // prepare the connection internal object, and reset channel if connection has been closed
     this.connections[url] = {
       conn: null,
-      channel: null
+      channels: { defaultChannel: null }
```

Comment:
```suggestion
      channels: { [defaultChannel]: null }
```
This should set the key of "DEFAULT_CHANNEL" in channels and not "defaultChannel"